djangocms_concurrent_users/templates/djangocms_concurrent_users/concurrency_button.html

```diff
@@ -19,3 +19,11 @@
 
 <link rel="stylesheet" href="{% static 'css/concurrent_users.css' %}">
 
+<div class="cms js-concurrentWarning concurrentWarning">
+    <div class="js-concurrentWarningMessage"></div>
+    <br>
+    <div class="cms-modal-item-buttons">
+        <a href="javascript:window.history.back();" type="button" class="cms-btn js-concurrentWarningButtonBack"></a>
```

Comment:
You were right, the href would have been overridden. Added a commit to fix that behavior.

> Why do you use JS here?
The scenario I wanted to introduce a back button is the following: 
userA edits pageA, userB edits pageB. When userA moves on to pageB which is blocked for him, since he is automatically in the edit mode for pageB. Click back should move him back to the edit mode of pageA.
Therefore using window.history.back() perfectly works. 
Do you have another suggestion?